presto-docs/src/main/sphinx/release/release-0.248.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Release 0.248
+=============
+
+**Highlights**
+==============
+* New aggregation function MAP_UNION_SUM.
+* New scalar function DESTRUCTURE_TDIGEST.
+* Support for session property managers, that can override session properties set by a user.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve output buffer memory tracking to reduce lock contention.
```

Comment:
Start with `Optimize` and also - how does this help ? Does this improve all queries or some specific set of queries?

Reply:
The release notes guideline state that all optimizations must start with "Improve". https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes
I've changed the phrasing to say "Improve query performance by reducing lock contention in output buffer memory tracking" for more clarity on the effect of the change